### PR TITLE
Best practices: add "enable 2FA" to the list

### PR DESCRIPTION
**Pull request by tripu**
h/t @SteveALee

### CODE REVIEW

File: best-practices.html [hunk 1]
```diff
@@ -29,6 +29,7 @@ <h1>Best practices and recommended tools</h1>
                 <li>
                     <a href="#all">For all GitHub users</a>
                     <ul>
+                        <li><a href="#2fa">Enable 2FA</a></li>
```
Comment (vivienlacourba) [inline best-practices.html]: s/Enable 2FA/Enable Two-factor authentication (2FA)/

This is to make it clearer for people that are not familiar with the acronym 2FA.

File: best-practices.html [hunk 2]
```diff
@@ -70,6 +71,15 @@ <h1>Best practices and recommended tools</h1>
         <main>
             <section id="all">
                 <h2>For all GitHub users <a href="#all">&#x1F517;</a></h2>
+                <section id="2fa">
+                    <h3>Enable 2FA <a href="#2fa">&#x1F517;</a></h3>
+                    <p>Multi-factor authentication, or two-factor authentication (<abbr title="Two-factor authentication">2FA</abbr>)
+                        is <a href="https://help.github.com/articles/about-two-factor-authentication/">a simple and efficient way to make our online identities
+                        and assets more robust against attacks</a>.
```
Comment (vivienlacourba) [inline best-practices.html]: I am not sure users would understand the benefit of 2FA for them when reading :
> to make our online identities and assets more robust against attacks

Here is a suggested update for this paragraph (taking text from the github about 2FA page and adding "to protect your online identity")

```html
<p><a href="https://help.github.com/articles/about-two-factor-authentication/">Two-factor authentication, or 2FA<a>, is an extra layer of security used when logging into websites or apps to protect your online identity. With 2FA, you have to log in with your username and password and provide another form of authentication that only you know or have access to.</p>
```

File: best-practices.html [hunk 3]
```diff
@@ -70,6 +71,15 @@ <h1>Best practices and recommended tools</h1>
...
+                        We encourage all users to enable 2FA in as many services and applications as they use for work
```
Comment (vivienlacourba) [inline best-practices.html]: s/ for work//

**Comment by tripu** [comment]
@vivienlacourba, thanks. All changed now. Please re-review and see if this resolves your three comments.

(When you are reviewing a PR and have small, specific changes to propose, there is [a cool feature, relatively new, called &ldquo;suggestions&rdquo;](https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/#starting-a-review) that is very handy because the author doesn't even need to use Git to accept/discard the suggestions: they can be incorporated into the PR with a click from the PR page itself.)

**Comment by tripu** [comment]
I just fixed a tiny markup error in @vivienlacourba's [suggestion](https://github.com/w3c/w3c.github.io/pull/58#discussion_r241687878), so I consider his review to be still valid, and I'm merging this.

**Pull request merged**